src_extra/networklayer_tcp.c

```diff
@@ -35,7 +35,6 @@
 
 /* workaround a glibc bug where an integer conversion is required */
 #if !defined(_WIN32)
-# include <features.h>
```

Comment:
@jpfr I tested compilation using travis and there I had the problem that features.h is not available on Mac. Maybe it's best to use a precompiler definition and include features.h only for linux.

Shall I change it back so we have a clean PR?